client/src/Components/Layouts/SearchPage/SearchPage.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useEffect } from 'react'
+import Header from '../../Common/Header/Header'
+import Navbar from '../../Common/Navbar/Navbar'
+import SearchIcon from '@material-ui/icons/Search'
+import grey from '@material-ui/core/colors/grey'
+import InputBase from '@material-ui/core/InputBase'
+import axios from 'axios'
+import { Grid } from '@material-ui/core'
+import ArtCard from '../../Common/ArtCard/ArtCard'
+import useStyles from './style'
+
+const SearchPage = () => {
+  const [searchValue, setValue] = useState('')
+  const [art, setArt] = useState(null)
+  const classes = useStyles()
+
+  useEffect(() => {
+    if (art) {
+      return
+    }
+    axios
+      .get('/api/get-art')
+      .then((result) => setArt(result.data))
+      .catch((err) => console.log(err))
+  }, [art])
+
+  const filterSearch = () => {
+    if (!searchValue) {
+      return art
+    }
+    return art.filter(
+      (artwork) =>
+        artwork.titel
+          .toLowerCase()
+          .includes(searchValue.toLowerCase()) ||
+        artwork.category
+          .toLowerCase()
+          .includes(searchValue.toLowerCase()),
+    )
```

Comment:
Might be fun to think about how to abstract this, so that the key is passed in. Bonus points for extracting the filter callback to its own function expression.